test/support.ts

```diff
@@ -228,7 +228,7 @@ export async function dropTestSchemas(sequelize: Sequelize) {
     }
   }
 
-  await Promise.all(schemasPromise.map(async p => p.catch((error: unknown) => error)));
```

Comment:
Is this change releated to DB2?

Reply:
Strictly speaking no, but it was muting errors and contributing to the flakiness